pkg/helm/charts/tests/centralservices/testdata/helmtest/scanner-v4.test.yaml

```diff
@@ -80,7 +80,8 @@ tests:
 - name: "Install mode is reported by Helm chart installation notes"
   tests:
   - name: "when installing neither indexer nor matcher"
-    scannerV4.disable: true
```

Comment:
This was a bug in the test.

Reply:
Looks like a but in helmtest too, if it does not complain on unknown fields? :thinking:

Reply:
true

Reply:
https://github.com/stackrox/helmtest/issues/83